packages/perseus/src/widgets/matcher/matcher.test.tsx

```diff
@@ -98,88 +102,96 @@ describe("matcher widget", () => {
         expect(container).toMatchSnapshot("moved items");
     });
 
-    it("can be answered correctly", async () => {
-        // Arrange
-        const apiOptions: APIOptions = {
-            isMobile: false,
-        };
-        const {renderer} = renderQuestion(question1, apiOptions);
-        await wait();
-
-        // Act
-        const matcher: Matcher = renderer.findWidgets("matcher 1")[0];
+    const answerfulItem = generateTestPerseusItem({question: question1});
+    const answerlessItem = splitPerseusItem(answerfulItem);
+
+    describe.each([
+        ["answerful", answerfulItem],
+        ["answerless", answerlessItem],
+    ])("given an %s item", (_, {question}) => {
+        it("can be answered correctly", async () => {
+            // Arrange
+            const apiOptions: APIOptions = {
+                isMobile: false,
+            };
+            const {renderer} = renderQuestion(question, apiOptions);
```

Comment:
I think `isMobile` defaults to false.

```suggestion
            const {renderer} = renderQuestion(question);
```